example/timer/odp_timer_accuracy.c

```diff
@@ -592,7 +746,10 @@ static void run_test(test_global_t *test_global)
 	timer_ctx_t *ctx;
 	test_stat_t *stat = &test_global->stat;
 	test_log_t *log = test_global->log;
-	int mode = test_global->opt.mode;
+	enum mode_e mode = test_global->opt.mode;
+	double period = ODP_TIME_SEC_IN_NS /
+			odp_fract_u64_to_dbl(&test_global->opt.freq) /
+			test_global->opt.multiplier;
```

Comment:
How about using fractional numbers until saving `tmo_ns` to avoid possible rounding errors?

Reply:
I figured double is probably enough. But if the significant bits of (ctx->count * period) don't fit into a double's significand, then we lose precision. Which is definitely possible with a high frequency and a long test.

I actually have another implementation which uses an integer fraction of tmo_ns, effectively forming a 64.63 bit fixed point tmo_ns. That code looked slightly less elegant to me.